composer.json

```diff
@@ -151,7 +151,7 @@
         "drupal/pathauto": "^1.12",
         "drupal/potion": "dev-2.x#e48d6ee336d960b9d2efb5d88e68402db6d229a6",
         "drupal/purge": "^3.6",
-        "drupal/recurring_events": "^2.0@RC",
+        "drupal/recurring_events": "^2",
```

Comment:
The change that updates the module has been reviewed here:

https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1486